### ISSUE TITLE
Gather list of hardware we have, and/or want.

### ISSUE THREAD

**Issue by leehambley** [leehambley/hhav#2]
A big deal, a decent video->sdcard capture is a big open question:

- https://www.startech.com/eu/AV/Converters/Video/standalone-video-capture-device~USB2HDCAPS looks good

For Mics I've always liked Røde and own three of their mics already

- http://www.rode.com/microphones/smartlav looks good, plus we'd need a wireless capture thing?

**Comment by scruplelesswizard** [comment]
Røde's are good. I have also had really good experiences with Shure when I used to work in theatre. They are the most durable I have used, and the sound quality is also quite good

**Comment by leehambley** [comment]
Good point :)

**Comment by christianhuening** [comment]
What camera do we have?

**Comment by leehambley** [comment]
So far only Go-Pros (which with the "anti-fisheye" post-processing work out really fine) but I'd like to find something better. 

Go Pros for timelapses and audience shots are totally adequate though.

Hardware list open questions is basically:

- standalone video capture from laptop (e.g between laptop and beamer, ideally to an SD card or SSD)
- decent camera (Confreaks used to use a Sony handicam)
- wireless (wearable) mics

**Comment by oliverlorenz** [comment]
I could provide have:
* Logitech c920
  * FullHD
  * Autofocus
* RODE Podcaster Microphone 
  * USB 
  * AUX
  * mounting
* Elgato GameCapture HD
  * 60fps @ FullHD
  * HDMI
* 1 Wireless Headset Microphone + Receiving Station
  * I have to look up what is the exactly name.
  * 3,5 clinc with adapter to 2,5
* A lot of cables
  * HDMI
  * USB Extenders
  * Audio

But to be honest: Webcam, RODE and GameCapture is also in private use. So we have to think about how we can handle this.

I always miss a second Wireless headset, because some talks are hold by two people.

**Comment by leehambley** [comment]
I have also a Røde Podcaster, which is not in a lot of use, and a Røde VideoMic Go (http://www.rode.com/microphones/videomicgo) which is sitting in a box since 1yr (used it a lot when we were pre-recording pitch videos and doing pitch training with my startup.

The GameCapture looks cool, but I've only been able to find one standalone device (records directly to removable media) - perhaps it would suffice to find cross-platform screen capture software and ask people to record their own laptop screens.  I have good experience with iShowU and iShowU Instant (https://shinywhitebox.com/ishowu-instant) for that. I suppose unlike gaming capture, there's not as much performance pressure, so we can easily rely on the host system to capture it's own output?

**Comment by oliverlorenz** [comment]
> I suppose unlike gaming capture, there's not as much performance pressure, so we can easily rely on the host system to capture it's own output?

Thats true if you speak in the primary use case of recording. I fully agree in that case. My equipment set is currently more focused on streaming, and capturing is a drive-by result. 

So usually I do live production. While the talk is running, I switching between prepared screens with static content, speaker total view and slides. I like this setup because I don't have to cut anything afterwards. Its instant available.

Here a little example: https://www.youtube.com/watch?v=F_K5S1NAZOU&list=PLhpmRI4kKiciy410nus97cZ1WITHaEXYu&index=3

As I understand you capturing different audio and video sources split and putting all together afterwards, correct?

Just some Ideas:
* Has somebody connections to the [c3voc](https://c3voc.de/)? Maybe they are open to share such dedicated recording hardware? 
* Maybe we can collect some money to buy such a recording hardware? Or a kind of sponsoring? I mean 235,99 € is a big amount for a single person, but not for a group of people or a sponsor

**Comment by scruplelesswizard** [comment]
From the perspective of a presenter I would have a hard time installing capture software on my machine, and especially for lightning talks or similar formats it would be hell to edit.

**Comment by leehambley** [comment]
Thanks to @chaosaffe for recommending a guy on twitter who had posted some stuff.

I link his tweet here and his notes on live production:

- https://twitter.com/the_sttts/status/1003599878289805312?s=03
- https://www.inovex.de/blog/capturing-and-live-streaming-presentations-a-non-intrusive-approach-tutorial/

Live Production puts more burden on everyone than "stuff to capture AV that people can use" - and it relies a bit on the skill of an operator at every usergroup.

I can't say now if it raises the bar for this idea too high, and/or the costs because a HHAV laptop and hardware seems to become more important as we dig deeper into this.

**Comment by leehambley** [comment]
sorry for radio silence all, I changed jobs and have been flat-out getting up to speed.
